front/app/containers/ChangePasswordPage/index.tsx

```diff
@@ -0,0 +1,209 @@
+import Button from 'components/UI/Button';
+import PasswordInput from 'components/HookForm/PasswordInput';
+import PasswordIconTooltip from 'components/UI/PasswordInput/PasswordInputIconTooltip';
+import { Helmet } from 'react-helmet';
+import ContentContainer from 'components/ContentContainer';
+import { FormLabel } from 'components/UI/FormComponents';
+import ChangePasswordSuccess from './ChangePasswordSuccess';
+
+// form
+import { useForm, FormProvider } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { string, object } from 'yup';
+
+// style
+import styled from 'styled-components';
+import messages from './messages';
+import { fontSizes, stylingConsts } from 'utils/styleUtils';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+
+// services
+import { changePassword } from 'services/users';
+import GetAppConfiguration, {
+  GetAppConfigurationChildProps,
+} from 'resources/GetAppConfiguration';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { handleHookFormSubmissionError } from 'utils/errorUtils';
+
+const StyledContentContainer = styled(ContentContainer)`
+  padding-bottom: 100px;
+`;
+
+const Title = styled.h1`
+  width: 100%;
+  color: #333;
+  font-size: ${fontSizes.xxxxl}px;
+  line-height: 40px;
+  font-weight: 500;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+  padding-top: 60px;
+  margin-bottom: 50px;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 20px;
+  margin-bottom: 10px;
+`;
+
+const Form = styled.form`
+  width: 100%;
+  max-width: 380px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-direction: column;
+`;
+
+const LabelContainer = styled.div`
+  display: flex;
+  align-items: center;
+  &.margin-top {
+    margin-top: 33px;
+  }
+`;
+
+const StyledPasswordIconTooltip = styled(PasswordIconTooltip)`
+  margin-bottom: 6px;
+`;
+
+type FormValues = {
+  current_password: string;
+  new_password: string;
+};
+
+type Props = {
+  tenant: GetAppConfigurationChildProps;
+};
+
+const ChangePassword = ({ tenant }: Props) => {
+  const { formatMessage } = useIntl();
+  const [success, setSuccess] = useState(false);
+
+  const minimumPasswordLength =
+    (!isNilOrError(tenant) &&
+      tenant.attributes.settings.password_login?.minimum_length) ||
+    0;
+
+  const schema = object({
+    current_password: string().required(
+      formatMessage(messages.currentPasswordRequired)
+    ),
+    new_password: string()
+      .test(
+        'length',
+        'Value can be empty or contain a string at with least the minimum password length',
+        (value, { createError, path }) => {
+          if (
+            value &&
+            value.length > 0 &&
+            value.length < minimumPasswordLength
+          ) {
+            return createError({
+              path,
+              message: formatMessage(messages.minimumPasswordLengthError, {
```

Comment:
I would use the copy from https://github.com/CitizenLabDotCo/citizenlab/blob/2859de9b8e90ea9d43947ef02c8f96f5b51ce84d/front/app/components/UI/PasswordInput/messages.ts#L67 to avoid duplication. But up to you.

Is it possible to use `front/app/components/UI/PasswordInput/index.tsx` here?

Reply:
yes the copy being used is the one you pointed out.
the password input component is the same also, im just using a wrapper for react form.